docs/source/roadmaps/may-2024.md

```diff
@@ -0,0 +1,117 @@
+(by reading and writing to [NeuroBlueprint](https://neuroblueprint.neuroinformatics.dev/) format)
+
+**No current ETA**
+
+- A common interface to prevalent pose estimation tools, and other machine-learning based object detection methods
+- A common interface to methods for behavioural segmentation and classification
+
+## Extracellular electrophysiology analysis
+**Current status**
+
+We are currently building [spikewrap](https://github.com/neuroinformatics-unit/spikewrap) (built on top of 
+[SpikeInterface](https://spikeinterface.readthedocs.io/en/latest/)) for simple, routine analysis of extracellular 
+electrophysiology analysis. For more details on this specific project, see the
+[spikewrap roadmap]().
+
+**Q2 2025**
+
+- Easy to use tool for all common extracellular electrophysiology analysis steps
+- Support for single experiments and complex multi-animal, multi-session datasets
+  (by reading and writing to [NeuroBlueprint](https://neuroblueprint.neuroinformatics.dev/) format)
+
+## Computational anatomy
+**Current status**
+
+Our computational anatomy work is based around the [BrainGlobe Initiative](https://brainglobe.info). BrainGlobe is
+centered around the [BrainGlobe Atlas API](https://brainglobe.info/documentation/brainglobe-atlasapi/index.html) which
+provides a common interface to multiple anatomical atlases. Currently, there are tools for:
+- 3D whole brain microscopy registration ([brainreg](https://brainglobe.info/documentation/brainreg/index.html))
+- 3D cell detection in large images ([cellfinder](https://brainglobe.info/documentation/cellfinder/index.html))
+- Analysis of arbitrary shaped structures in atlas space such as lesions and implanted devices
+  ([brainglobe-segmentation](https://brainglobe.info/documentation/brainglobe-segmentation/index.html))
+- Visualisation of atlas registered data ([brainrender](https://brainglobe.info/documentation/brainrender/index.html) &
+  [brainglobe-heatmap](https://brainglobe.info/documentation/brainglobe-heatmap/index.html))
+- Various small tasks such as [reorienting anatomical images](https://brainglobe.info/documentation/brainglobe-space/index.html)
+
+**Future**
+
+BrainGlobe is a large project with many short- and long-term aims. Our two main goals are to:
+- Allow for all neuroscience histology analysis and visualisation tasks to be carried out easily in a single
+  environment. This includes 2D & 3D data in all common animal models
+- Create an end-to-end pipeline to analyse mouse brains imaged with a [mesoSPIM](https://mesospim.org/)
+
+For full details, please see the [BrainGlobe roadmap](https://brainglobe.info/community/roadmaps/index.html).
+
+## Multimodal integration
+**Current status**
+
+Currently, our data analysis tools are restricted to a single modality. It is important that multimodal data can be 
+combined to e.g. correlate neural activity and behaviour. There are some excellent existing packages for neurophysiological 
```

Comment:
```suggestion
combined at the analysis stage e.g. to correlate neural activity and behaviour. There are some excellent existing packages for neurophysiological 
```

Reply:
or following preprocessing? happy for suggestion to be rejected not 100% on this.